orttraining/orttraining/python/training/ortmodule/torch_cpp_extensions/cuda/torch_gpu_allocator/torch_gpu_allocator.cc

```diff
@@ -1,19 +1,30 @@
 // Copyright (c) Microsoft Corporation. All rights reserved.
 // Licensed under the MIT License.
 
-#include <torch/extension.h>
 #include <c10/___gpu_identifier___/___gpu_allocator_header___.h>
+#include <torch/extension.h>
+
+void* delegate_raw_alloc(size_t nbytes) {
+  auto allocator = c10::___gpu_identifier___::___gpu_allocator_header___::get();
```

Comment:
How costly is it to get the allocator every time? 
Do we need to cache it?

Reply:
No. It's just a pointer.